_includes/specials/policy/fr/02-utilite-du-cache.md

```diff
@@ -0,0 +1,57 @@
+
+Le mempool de Bitcoin Core calcule le taux de frais de chaque entrée et de ses ascendants (appelé _ancestor feerate_), met en cache ce
+résultat et utilise un algorithme de construction de modèle de bloc avide. Il trie le mempool dans l'ordre du _score d'ancêtre_ (le
+minimum du taux de frais de l'ascendant et du taux de frais individuel) et sélectionne les paquets d'ascendants dans cet ordre, en mettant à jour les informations sur les frais et le poids des ascendants des transactions restantes au fur et à mesure. Cet algorithme offre un équilibre
+entre performance et rentabilité et ne produit pas nécessairement des résultats optimaux. Son efficacité peut être améliorée en
+limitant la taille des transactions et des paquets ascendants---Bitcoin Core fixe ces limites à 400 000 et 404 000 unités de poids,
+respectivement.
+
+De même, un _descendant score_ est calculé et utilisé lors de la sélection des paquets à expulser du mempool, car il serait
+désavantageux d'expulser une transaction à faible taux de frais qui a un descendant à fort taux de frais.
+
+La validation du Mempool fait également appel à des frais et à des tarifs lorsqu'il s'agit de transactions qui dépensent les mêmes
+intrants, c'est-à-dire des transactions à double dépense ou des transactions conflictuelles. Au lieu de toujours conserver la
+première transaction qu'ils voient, les nœuds utilisent un ensemble de règles pour déterminer quelle transaction est la plus
+incitative à conserver. Ce comportement est connu sous le nom de [Replace by Fee][topic rbf].
+
+Il est intuitif que les mineurs maximisent les frais, mais pourquoi un opérateur de nœud non-mineur devrait-il mettre en œuvre ces
+politiques ? Comme indiqué dans le billet de la semaine dernière, l'utilité du pool de mémoire d'un nœud non-mineur est directement
+liée à sa similarité avec les pools de mémoire des mineurs. Ainsi, même si un opérateur de nœud n'a jamais l'intention de produire
+un bloc en utilisant le contenu de son mempool, il a intérêt à conserver les transactions les plus compatibles avec les incitations.
+
+Bien qu'il n'y ait pas de règles consensuelles exigeant que les transactions paient des frais, les frais et le taux de frais jouent un
+rôle important dans le réseau Bitcoin en tant que moyen "équitable" de résoudre la concurrence pour l'espace des blocs. Les mineurs
+utilisent le taux de frais pour évaluer l'acceptation, l'expulsion et les conflits, tandis que les nœuds non mineurs suivent ces
+comportements afin de maximiser l'utilité de leurs mempools.
+
+La rareté de l'espace des blocs exerce une pression à la baisse sur la taille des transactions et encourage les développeurs à être
+plus efficaces dans la construction des transactions. Le billet de la semaine prochaine explorera des stratégies et des techniques
+pratiques pour minimiser les frais dans les transactions sur la chaîne.
+
+[policy01]: /fr/newsletters/2023/05/17/#en-attente-de-confirmation-1-pourquoi-avons-nous-un-mempool-
```

Comment:
```suggestion
[policy01]: /fr/newsletters/2023/05/17/#en-attente-de-confirmation-1--pourquoi-avons-nous-un-mempool-
```

If you navigate to https://bitcoinops.org/fr/newsletters/2023/05/17/ and then right click on the "En attente de confirmation #1 : pourquoi avons-nous un mempool ?" header in your browser and inspect, you can see the ID="en-attente-de-confirmation-1--pourquoi-avons-nous-un-mempool-"

Reply:
You are the best ! :-) 

Thanks